checkQC/config.py

```diff
@@ -97,10 +98,7 @@ def _get_matching_handler(self, instrument_and_reagent_type, read_length):
             else:
                 if int(read_length) == int(config_read_length):
                     return self._config[instrument_and_reagent_type][int(config_read_length)]["handlers"]
-        raise ConfigEntryMissing("Could not find a config entry for instrument '{}' "
-                  "with read length '{}'. Please check the provided config "
-                  "file ".format(instrument_and_reagent_type,
-                                 read_length))
+        raise ConfigEntryMissing
```

Comment:
No need to remove the message here. It's great to have when debugging.

Reply:
Ah ok, I'll restore it.